diff -rupN --no-dereference numpy-1.24.1/numpy/distutils/ccompiler_opt.py numpy-1.24.1-new/numpy/distutils/ccompiler_opt.py
--- numpy-1.24.1/numpy/distutils/ccompiler_opt.py	2022-12-26 04:52:52.998270000 +0100
+++ numpy-1.24.1-new/numpy/distutils/ccompiler_opt.py	2023-01-28 23:04:55.146165789 +0100
@@ -203,7 +203,7 @@ class _Config:
             # native usually works only with x86
             native = '-march=native',
             opt = '-O3',
-            werror = '-Werror',
+            werror = '',
         ),
         clang = dict(
             native = '-march=native',
