diff -rupN --no-dereference numpy-1.23.5/numpy/distutils/ccompiler_opt.py numpy-1.23.5-new/numpy/distutils/ccompiler_opt.py
--- numpy-1.23.5/numpy/distutils/ccompiler_opt.py	2022-11-19 23:40:58.105490400 +0100
+++ numpy-1.23.5-new/numpy/distutils/ccompiler_opt.py	2022-12-05 10:10:52.848488207 +0100
@@ -203,7 +203,7 @@ class _Config:
             # native usually works only with x86
             native = '-march=native',
             opt = '-O3',
-            werror = '-Werror',
+            werror = '',
         ),
         clang = dict(
             native = '-march=native',
