diff -rupN --no-dereference numpy-1.18.2/numpy/random/src/mt19937/mt19937.h numpy-1.18.2-new/numpy/random/src/mt19937/mt19937.h
--- numpy-1.18.2/numpy/random/src/mt19937/mt19937.h	2020-02-15 17:17:33.000000000 +0100
+++ numpy-1.18.2-new/numpy/random/src/mt19937/mt19937.h	2020-03-18 17:39:43.275537616 +0100
@@ -2,7 +2,7 @@
 #include <math.h>
 #include <stdint.h>
 
-#ifdef _WIN32
+#ifdef _MSC_VER
 #define inline __forceinline
 #endif
 
diff -rupN --no-dereference numpy-1.18.2/numpy/random/src/pcg64/pcg64.h numpy-1.18.2-new/numpy/random/src/pcg64/pcg64.h
--- numpy-1.18.2/numpy/random/src/pcg64/pcg64.h	2020-02-15 17:17:33.000000000 +0100
+++ numpy-1.18.2-new/numpy/random/src/pcg64/pcg64.h	2020-03-18 17:39:43.276537605 +0100
@@ -52,7 +52,7 @@
 
 #include <inttypes.h>
 
-#ifdef _WIN32
+#ifdef _MSC_VER
 #include <stdlib.h>
 #define inline __forceinline
 #endif
diff -rupN --no-dereference numpy-1.18.2/numpy/random/src/philox/philox.h numpy-1.18.2-new/numpy/random/src/philox/philox.h
--- numpy-1.18.2/numpy/random/src/philox/philox.h	2020-02-15 17:17:33.000000000 +0100
+++ numpy-1.18.2-new/numpy/random/src/philox/philox.h	2020-03-18 17:39:43.276537605 +0100
@@ -33,7 +33,7 @@ static NPY_INLINE uint64_t mulhilo64(uin
   return (uint64_t)product;
 }
 #else
-#ifdef _WIN32
+#ifdef _MSC_VER
 #include <intrin.h>
 #if defined(_WIN64) && defined(_M_AMD64)
 #pragma intrinsic(_umul128)
