diff -rupN --no-dereference numpy-1.24.1/numpy/core/setup.py numpy-1.24.1-new/numpy/core/setup.py
--- numpy-1.24.1/numpy/core/setup.py	2022-12-26 04:52:52.930270000 +0100
+++ numpy-1.24.1-new/numpy/core/setup.py	2023-01-28 23:04:54.787161090 +0100
@@ -81,9 +81,7 @@ def can_link_svml():
     if NPY_DISABLE_SVML:
         return False
     platform = sysconfig.get_platform()
-    return ("x86_64" in platform
-            and "linux" in platform
-            and sys.maxsize > 2**31)
+    return False
 
 def check_svml_submodule(svmlpath):
     if not os.path.exists(svmlpath + "/README.md"):
diff -rupN --no-dereference numpy-1.24.1/numpy/random/src/pcg64/pcg64.h numpy-1.24.1-new/numpy/random/src/pcg64/pcg64.h
--- numpy-1.24.1/numpy/random/src/pcg64/pcg64.h	2022-12-26 04:52:53.070270300 +0100
+++ numpy-1.24.1-new/numpy/random/src/pcg64/pcg64.h	2023-01-28 23:04:54.788161103 +0100
@@ -52,7 +52,7 @@
 
 #include <inttypes.h>
 
-#ifdef _WIN32
+#ifdef _MSC_VER
 #include <stdlib.h>
 #endif
 
